synphot/filter_parameterization/filter_fft.py

```diff
@@ -73,6 +84,9 @@ def filter_to_fft(bp, wavelengths=None, n_terms=10):
     return n_lambda, lambda_0, delta_lambda, tr_max, fft.tolist()
 
 
+# TODO: Option to construct Sine1D compound model instead of empirical?
+#       See implementation in tynt/core.py for model=True. It was not ported
+#       due to https://github.com/bmorris3/tynt/issues/9
```

Comment:
Linking this line to updated discussion in https://github.com/bmorris3/tynt/issues/9.

Reply:
Update: I added back the analytical model and documented the caveat of sampling requirement. No point throwing away the work done, but also no need to fix anything right now. Thanks!